src/utils/domUtils.ts

```diff
@@ -9,99 +9,58 @@ export module Utils {
      * @param {d3.Selection} element
      * @returns {SVGRed} The bounding box.
      */
-    export function getBBox(element: d3.Selection<any>): SVGRect {
+    export function getBBox(element: d3.Selection<any>) {
```

Comment:
Are we keeping `getBBox()`?  It reflects the endpoint name, but it's inconsistent now.  Not sure...

Maybe `elementBBox()`?